pkg/addon/cache_test.go

```diff
@@ -31,3 +31,63 @@ func TestPutVersionedUIData2cache(t *testing.T) {
 	assert.NotEmpty(t, u.versionedUIData["helm-repo"]["fluxcd-1.0.0"])
 	assert.Equal(t, u.versionedUIData["helm-repo"]["fluxcd-1.0.0"].Name, "fluxcd")
 }
+
+func TestPutAddonUIData2Cache(t *testing.T) {
```

Comment:
How is this different from `TestPutVersionedUIData2cache`?

Reply:
These test different func in cache.go